src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,47 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int, ring::Union{MPolyRing, Nothing} = nothing)
+
+Given an ambient dimension and a subspace dimenson dimension return the ideal
+generated by the plucker relations. If the ring is not specified return the ideal in the multi variate polynomial ring over the rationals.
+
+# Examples
+```jldoctest
+julia> R, x = PolynomialRing(ResidueRing(ZZ, 7), "x" => (1:2, 1:3), ordering=:degrevlex)
+(Multivariate Polynomial Ring in 6 variables x[1, 1], x[2, 1], x[1, 2], x[2, 2], ..., x[2, 3] over Integers modulo 7, nmod_mpoly[x[1, 1] x[1, 2] x[1, 3]; x[2, 1] x[2, 2] x[2, 3]])
+
+julia> grassman_pluecker_ideal(2, 4; ring = R)
+ideal(x[1, 2]*x[2, 2] + 6*x[2, 1]*x[1, 3] + x[1, 1]*x[2, 3])
+```
+"""
+function grassman_pluecker_ideal(subspace_dimension::Int,
+                                 ambient_dimension::Int;
+                                 ring::R = nothing) where R <: Union{MPolyRing, Nothing}
+    pluecker_ideal = convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
+    if R != nothing
+        converted_generators = []
+        for g in gens(pluecker_ideal)
+            coeff_ring = base_ring(ring)
+            converted_coeffs = [coeff_ring(numerator(c)) for c in coefficients(g)]
+            polynomial = MPolyBuildCtx(ring)
+
+            for (i, exponent) in enumerate(exponent_vectors(g))
+                c = converted_coeffs[i]
+                push_term!(polynomial, c, exponent)
+            end
+            converted_g = finish(polynomial)
+            push!(converted_generators, converted_g)
+        end
+        pluecker_ideal = ideal(ring, converted_generators)
+    end
```

Comment:
Actually, can't you just do
```suggestion
    pluecker_ideal = convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension), parent = ring)
```

Reply:
I had this at first, but the issue is that convert deals with polymake polynomial ideals which are over the rationals,
and so i need to do the parsing from rationals to some other coefficient ring in this specific case here, so that convert still works outside of this case

Reply:
we are working on changing this in polymake, and this can eventually be removed,
here is the polymake ticket https://trac.polymake.org/ticket/1270#ticket